src/test/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManagerSpec.groovy

```diff
@@ -219,12 +244,14 @@ class LSFJobManagerSpec extends Specification {
         jobInfo.jobName == "ls -l"
         jobInfo.tool == new File("ls -l")
         jobInfo.jobID == new BEJobID("22005")
-        jobInfo.submitTime == ZonedDateTime.of(2017, 12, 28, 19, 56, 0, 0, ZoneId.systemDefault())
+
+        // The year-parsing/inferrence is checked in another test. Here just take the parsed value.
+        jobInfo.submitTime == ZonedDateTime.of(jobInfo.submitTime.year, 12, 28, 19, 56, 0, 0, ZoneId.systemDefault())
         jobInfo.eligibleTime == null
-        jobInfo.startTime == ZonedDateTime.of(2017, 12, 28, 19, 56, 0, 0, ZoneId.systemDefault())
-        jobInfo.endTime == ZonedDateTime.of(2017, 12, 28, 19, 56, 0, 0, ZoneId.systemDefault())
-        jobInfo.executionHosts == ["tbi-cn019", "tbi-cn019"]
-        jobInfo.submissionHost == "tbi-cn013"
+        jobInfo.startTime == ZonedDateTime.of(jobInfo.submitTime.year, 12, 28, 19, 56, 0, 0, ZoneId.systemDefault())
```

Comment:
Also all three times use a similar instances of time objects. Create once and reuse.

Reply:
I only modified the existing tests.